### ISSUE TITLE
Pages sometimes get cut off when JS is disabled

### ISSUE THREAD

**Issue by abizer** [ocf/ocfweb#278]
I don't remember what this is called but for longer docs the bottom part gets hidden by the ocfweb footer. This occurs on Chrome in OS X and Chromium on Linux. See example:

![screen shot 2017-04-05 at 2 38 16 pm](https://cloud.githubusercontent.com/assets/1916822/24728201/d8a0fc42-1a0d-11e7-867a-8f047cff4b83.png)

**Comment by chriskuehl** [comment]
This isn't supposed to happen and doesn't happen for me. Can you provide more information about browser / OS / JS / etc?

https://www.ocf.berkeley.edu/docs/staff/procedures/vhost/

**Comment by matthew-mcallister** [comment]
This happens if you disable JS for berkeley.edu. See for yourself.

**Comment by tahabi** [comment]
yeah, Matt is right. Presumably we shouldn't have to enable JS to use the docs, I'll see if I can find a way to fix it.

**Comment by matthew-mcallister** [comment]
This affects every page, not just docs. If you find a fix, that'd be great.

**Comment by chriskuehl** [comment]
yes, that's expected

**Comment by chriskuehl** [comment]
patches still welcome, but it's almost impossible to implement a sticky footer correctly in a compatible way without JS. you can probably improve this case though.

**Comment by redplanks** [comment]
Flexbox?

**Comment by chriskuehl** [comment]
that was "compatible way" comment, though maybe we're okay with it not working on really old browsers -- it's just the footer, after all (plus, flexbox has okay support now -- 88% or 97% depending on if you're okay with yucky prefixes)

**Comment by jvperrin** [comment]
We can probably auto-generate all the prefixes too, there's some things that do that (like [autoprefixer](https://github.com/postcss/autoprefixer) for instance), so that seems reasonable in support.
